src/dmd/initsem.d

```diff
@@ -604,6 +610,19 @@ extern(C++) Initializer initializerSemantic(Initializer init, Scope* sc, ref Typ
             return err();
         }
 
+        /* If it's an array of integral being initialized by `{ string }`
+         * replace with `string`
+         */
+        auto tn = t.nextOf();
+        if (tn.isintegral())
```

Comment:
This is the single use of `tn`. Rewrite to `if (t.nextOf().isintegral())`. Also, why is `isintegral` not respecting the camel case convention?

Reply:
1. tn is used later
2. isintegral() is very old, and refactoring that is off topic to this PR